impeller/renderer/backend/vulkan/gpu_tracer_vk.h

```diff
@@ -47,20 +38,34 @@ class GPUTracerVK {
   bool IsEnabled() const;
 
  private:
+  friend struct GPUProbe;
+
+  static const constexpr size_t kTraceStatesSize = 32u;
```

Comment:
I increased the size of the number of pending queries because on very slow apps I observed that we would get backed up more than the previous 16.